media-gfx/sane_backends/sane_backends-1.0.25.recipe

```diff
@@ -81,21 +82,18 @@ BUILD_PREREQUIRES="
 
 BUILD()
 {
-	libtoolize --force --copy --install
-	aclocal
-	autoconf
-	automake
-
 	runConfigure configure \
 		--prefix=$prefix \
 		--bindir=$binDir \
 		--sbindir=$binDir \
 		--libdir=$libDir \
 		--includedir=$includeDir \
 		--datadir=$dataDir \
-		--with-docdir=$docDir \
+		--docdir=$docDir \
 		--enable-pthread \
+		--disable-ipv6 \
```

Comment:
This is definitely wrong, we don't support it really yet.

Reply:
what do you mean with " we don't support it really yet"?